rest/package.json

```
@@ -10,41 +10,40 @@
     "start": "node _build/index.js",
     "start:debug": "node src/index.js",
     "test": "mocha --full-trace --recursive",
-    "test:coverage": "nyc yarn test && nyc report --reporter=text-lcov",
+    "test:coverage": "nyc npm test && nyc report --reporter=text-lcov",
     "test:jenkins": "cross-env JUNIT_REPORT_PATH=test-results.xml mocha --reporter mocha-jenkins-reporter --mongoHost db --forbid-only --full-trace --recursive test || exit 0",
-    "test:travis": "nyc npm run test && nyc report --reporter=text-lcov | coveralls",
+    "test:travis": "nyc npm test && nyc report --reporter=text-lcov | coveralls",
     "lint": "eslint src test",
     "lint:jenkins": "eslint -o tests.catapult.lint.xml -f junit src test || exit 0"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "chai": "^4.1.2",
-    "cross-env": "^5.1.3",
-    "eslint": "^4.18.2",
-    "eslint-config-airbnb": "^16.1.0",
-    "eslint-plugin-import": "^2.9.0",
-    "eslint-plugin-jsx-a11y": "^6.0.3",
-    "eslint-plugin-react": "^7.7.0",
-    "eslint-plugin-sort-requires": "^2.1.0",
-    "hippie": "^0.5.1",
+    "chai": "^4.2.0",
+    "cross-env": "^5.2.0",
+    "eslint": "^6.0.1",
+    "eslint-config-airbnb": "^17.1.1",
+    "eslint-plugin-import": "^2.18.0",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-react": "^7.14.2",
+    "hippie": "^0.5.2",
     "minimist": "^1.2.0",
-    "mocha": "^5.0.1",
-    "mocha-jenkins-reporter": "^0.3.10",
+    "mocha": "^6.1.4",
+    "mocha-jenkins-reporter": "^0.4.1",
     "ncp": "^2.0.0",
-    "rimraf": "^2.6.2",
-    "sinon": "^7.2.2"
+    "rimraf": "^2.6.3",
+    "sinon": "^7.3.2"
   },
   "dependencies": {
     "catapult-sdk": "link:../catapult-sdk",
```

```suggestion
     "catapult-sdk": "file:../catapult-sdk",
```
Can we change _link_ for _file_? That would stop npm incompatibilities. For example to use `npm outdated` tool.

---

related: https://slack.nem.ninja/gap-dev/pl/oieh3e1ku3djudejgxm9m9g7no

i remember i made this suggstion/change months ago as well, and it was rejected, though i believe having `file` here would be better

---

merging anyway because this is not entirely related to this PR